CHANGELOG-developer.next.asciidoc

```diff
@@ -95,6 +96,7 @@ The list below covers the major changes between 7.0.0-rc2 and main only.
 
 ==== Added
 
+- Add new fields to configure the lease duration, retry and renew when using leader elector with kubernetes autodiscover.{pull}38471[38471]
```

Comment:
this should be part of the public changelog I think, not the developer one -
https://github.com/elastic/beats/blob/main/CHANGELOG.asciidoc#L49

Reply:
Ok. I moved it to changelog.next, is it ok now?